### ISSUE TITLE
Emacs Eclim not recognizing project

### ISSUE THREAD

**Issue by howdoicomputer** [emacs-eclim/emacs-eclim#16]
So I'm able to open up my project in Eclipse, get it to run, etc. It is a Maven project that I've imported.

It looks like I can use the eclim CLI to import the project...

```
C:\tools\eclipse
λ ./eclim -command project_import -f C:/Users/drfrankinfurter/workspace/democratizeddata
"Project with name \u0027democratizeddata\u0027 already exists at \u0027C:/Users/drfrankinfurter/workspace/democratizeddata\u0027."
```

However, when I try working on the project from within Emacs I get an `invalid project` error message. I've tried importing the project using emacs-eclim but the function just executes without returning anything.

My workspace *is* recognized so that doesn't seem to be the issue.

I'm running Windows 10 and Emacs 25 with Spacemacs on top of it. I'm in the java layer.

Really, all I want is Spring Boot hot reloading.

**Comment by nloyola** [comment]
Try this:

1. Open one of your project Java files in Emacs,
1. Toggle debug messages for Eclim: <kbd>M-x eclim-toggle-print-debug-messages</kbd>,
1. Start the Eclim server: <kbd>start-eclimd</kbd>.

Then paste the contents of the `*Messages*` buffer here, starting from when you opened the Java file.

**Comment by howdoicomputer** [comment]
@nloyola Thanks for chiming in!

However, I got it working. The problem was that I wasn't escaping my Windows paths like so:

```
  (setq eclim-eclipse-dirs "C:\\tools\\eclipse"
        eclim-executable "C:\\tools\\eclipse\\eclim.bat")
        eclimd-default-workspace "C:\\Users\\drfrankinfurter\\workspace")
```

However, I did come across another problem. If emacs-eclim waits for the eclimd process on Windows then emacs will be blocked perpetually - even if eclimd comes up successfully. Not a big deal since I can just set `(setq eclimd-wait-for-process nil)` to false but it's something to note.
